testing/scripts/test_helpdesk_pw.py

```
@@ -195,8 +196,69 @@ def skip(name: str, reason: str = ""):
+            if resp.status_code != 200:
+                return []
+            matched = [
+                m for m in resp.json().get("items", [])
+                if any(
+                    to_email.lower() in (r.get("Mailbox", "") + "@" + r.get("Domain", "")).lower()
+                    for r in m.get("To", [])
+                )
+            ]
+            if matched:
+                return matched
+        except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)